src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionActions.tag

```diff
@@ -3,22 +3,20 @@
 <%@ taglib tagdir="/WEB-INF/tags/instructor/feedbacks" prefix="tif" %>
 <%@ tag import="teammates.common.util.Const" %>
 <%@ attribute name="actions" type="teammates.ui.template.InstructorFeedbackSessionActions" required="true" %>
-<a class="btn btn-default btn-xs btn-tm-actions session-edit-for-test margin-bottom-7px"
+<a class="btn btn-default btn-xs btn-tm-actions session-edit-for-test margin-bottom-7px<c:if test="${not actions.allowedToEdit}"> disabled</c:if>"
     href="${actions.editLink}"
     title="<%= Const.Tooltips.FEEDBACK_SESSION_EDIT %>"
     data-toggle="tooltip"
-    data-placement="top"
-    <c:if test="${not actions.allowedToEdit}">disabled</c:if>>
```

Comment:
Hi friends just want to clarify (as a noob to jsp): so based on my understanding, the thing enclosed inside `${}` should be some object passed into this tag, and thus the `allowedToEdit` is one of the attributes of this `actions` enclosed inside `${}`. However based on my tracing, this action eventually goes back to `InstructorFeedbackSessionActions`, which has `editLink`, `remindLink`... and `isAllowedToEdit` as contrary to the `allowedToEdit` as accessed in the `${}` here. So can someone help me clarify where this `allowedToEdit` comes from? or is my understanding of jsp wrong? thx!

Reply:
That's a JavaBean, so `allowedToEdit` is matched by the pattern `isAllowedToEdit`